src/utils/helpers.ts

```diff
@@ -46,3 +46,6 @@ export const formatNumber = (
  */
 export const formatInteger = (amount: number | string) =>
   formatNumber(amount, { fractionDigits: 0 });
+
+export const isBrowser = () =>
```

Comment:
I think vueuse already has such a utility which is named `isClient` can you check?

Reply:
done.